components/dashboard/src/provider-utils.tsx

```diff
@@ -56,12 +56,8 @@ async function openAuthorizeWindow(params: OpenAuthorizeWindowParams) {
             search: `returnTo=${encodeURIComponent(returnTo)}&host=${host}&override=true&scopes=${(scopes || []).join(',')}`
         }).toString();
 
-    const newWindow = window.open(url, "gitpod-auth-window");
-    if (!newWindow) {
```

Comment:
reverting this to get to square one, where the untracked new window was opened just fine. 

let's think this through one more time, next time 🙏🏻